src/openforms/js/components/admin/form_design/FormStepDefinition.js

```diff
@@ -253,15 +281,11 @@ FormStepDefinition.propTypes = {
   errors: PropTypes.arrayOf(PropTypes.arrayOf(PropTypes.string)),
 };
 
-const ConfigurationErrors = ({errors = []}) => {
-  const configurationErrors = errors.filter(([name, err]) => name === 'configuration');
-  const [hasConfigurationErrors, formattedConfigurationErrors] =
-    normalizeErrors(configurationErrors);
-
-  if (!hasConfigurationErrors) return null;
+const ConfigurationErrorList = ({errors = []}) => {
```

Comment:
Why not use https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/admin/forms/ErrorList.js#L4 ?

Reply:
I didn't realize that's (almost) the same! I saw it because in this file there was some manual error list building and figured we didn't have anything for it yet.

I'll update to use the re-usable component.

Reply:
In the end we needed `MessageList` component rather than `ErrorList` because of the classnames involved, and I saw that we've been doing some funky stuff with that anyway so I refactored it. Now we are only using existing components.